DigitalExcellence/dex-backend
# Feature/522 sort out inactive projects from search result

## Description

A few sentences describing the overall goals of the pull request's commits.

## Type of change


## Checklist


## Steps to Test or Reproduce
1. Start the dex backend project.
2. Log in as an administrator.
3. After you have logged in, check the two new Endpoints. 
4. Try to update the multipliers and then check if the response of the post request is adjusted as well.
## Link to issue
[#522]
Closes: #522
